routes/mountains.js

```diff
@@ -0,0 +1,191 @@
+const db = require('../db');
+const express = require('express');
+app = express.Router();
+
+app.use(express.static('public'));
+
+const head = `
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>The Fourteeners</title>
+<link rel="stylesheet" href="./style.css" />
+<script src="https://kit.fontawesome.com/dc998fff98.js" crossorigin="anonymous"></script>
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Permanent+Marker&display=swap" rel="stylesheet">
+</head>
+`;
+
+const navBar = `
+<div id="nav">
+<ul id="sourceInfo">
+  <li>Project by Stephen Wong at Full Stack Academy</li>
+  <li style="float:right"><span>Data Source: <a href="https://www.14ers.com/" target="_blank">14ers.com</a></span></li>
+</ul>
+<ul>
+  <li class="link" style="float:right"><a class="active" href="https://github.com/stephen-wong-tx/dealers_choice_js" target="_blank">About</a></li>
+  <li id="icon"><a href="/"><img src="./fourteeners-home-alt.png" alt="The number 14 in front of a mountain range drawing"></a></li>
+</ul>
+</div>
+`;
+
+app.get('/', async(req, res, next) => {
+  try{
+    const ranges = await db.models.Range.findAll();
+
+    let html = `
+    <!DOCTYPE html>
+    <html lang="en">
+      ${head}
+      <body>
+        ${navBar}
+        <div id="main-content">
+        <h1>The Fourtneeners</h1>
+        <h2 style="font-size: .8rem">Sequelize & PostgreSQL Version</h2>
+        <p class="description"> <strong>Fourteener</strong> fȯr-ˈtēn-ər<br /> A mountain peek with an elevation of at least 14,000 ft (4267 m).</p>
+          <div id="entry-list">
+            ${ranges.map( range => `
+              <div class="entryContainer">
+                <h2>${range["Name"]}</h2>
+                <div id="button-${range.ID}" class="details-container"><a href="/mountains/${range.ID}">See details</a></div>
+              </div>
+            `).join("")}        
+            
+          </div>
+        </div>
+      </body>
+    </html>
+    `
+    res.send(`
+    ${html}
+    `
+    );
+  }
+  catch(error) {
+    next(error);
+  }
+})
+
+app.get('/:range', async(req, res, next) => {
+  try {
+    const range = await db.models.Mountain.findAll(
+      {
+        where : {
+          RangeID: req.params.range
+        }
+      }
+    )
```

Comment:
good place to use associations like Mountain.belongTo(Range) and Range.hasMany(Mountain) but i know we havent learned sequelize at this point so it's all good